packages/perseus-editor/src/components/locked-point-settings.tsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * LockedPointSettings is a component that allows the user to edit the
+ * settings of specifically a locked point on the graph.
+ *
+ * Used in the interactive graph editor's locked figures section.
+ */
+import {View, useUniqueIdWithMock} from "@khanacademy/wonder-blocks-core";
+import {TextField} from "@khanacademy/wonder-blocks-form";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import {Spring} from "@khanacademy/wonder-blocks-layout";
+import {color, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {LabelMedium, LabelLarge} from "@khanacademy/wonder-blocks-typography";
+import trashIcon from "@phosphor-icons/core/bold/trash-bold.svg";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import {getValidNumberFromString} from "./util";
+
+import type {LockedPoint} from "@khanacademy/perseus";
+
+export type Props = LockedPoint & {
+    onRemove: () => void;
+    onChangeCoord: (coord: [number, number]) => void;
+};
+
+const LockedPointSettings = (props: Props) => {
+    const {coord, onRemove, onChangeCoord} = props;
+    const [coordState, setCoordState] = React.useState([
+        // Using strings to make it easier to work with the text fields.
+        coord[0].toString(),
+        coord[1].toString(),
+    ]);
+
+    // Generate unique IDs so that the programmatic labels can be associated
+    // with their respective text fields.
+    const ids = useUniqueIdWithMock();
+    const xCoordId = ids.get("x-coord");
+    const yCoordId = ids.get("y-coord");
+
+    function handleBlur() {
+        const validCoord = [
+            getValidNumberFromString(coordState[0]),
+            getValidNumberFromString(coordState[1]),
+        ] as [number, number];
+
+        // Make the text field only show valid numbers after blur.
+        setCoordState([validCoord[0].toString(), validCoord[1].toString()]);
+        // Update the graph with the new coordinates.
+        onChangeCoord(validCoord);
+    }
+
+    function handleChange(newValue, coordIndex) {
+        const newCoord = [...coordState];
+        newCoord[coordIndex] = newValue;
+        setCoordState(newCoord);
+    }
+
+    return (
+        <View style={styles.container}>
+            {/* Title row */}
+            <View style={styles.row}>
+                <LabelLarge>Point</LabelLarge>
+                <Spring />
+                <IconButton
+                    icon={trashIcon}
+                    aria-label={`Delete locked point at ${coordState[0]}, ${coordState[1]}`}
+                    onClick={onRemove}
+                />
+            </View>
+
+            {/* Coordinates */}
+            <View>
+                <View style={styles.row}>
```

Comment:
**thought**: From a design perspective, I wonder if we should put the x and y coord inputs on the same row.  Vertical space is so precious in our editors and so I think we should try to avoid scrolling if at all possible.  

What do you think about something like this: 

```
*Point* 
Coordinate [x ] [y ]
```

Where the inputs would have a watermark/placeholder for which axis they're for?  I think authors using this editor would understand "x" always comes first so even with data in the fields they'd likely understand it's an x, y coordinate pair.